apps/web/pages/api/social/og/image.tsx

```diff
@@ -63,59 +62,63 @@ export default async function handler(req: NextApiRequest) {
     ] as SatoriOptions["fonts"],
   };
 
-  switch (imageType) {
-    case "meeting": {
-      const { names, usernames, title, meetingProfileName, meetingImage } = meetingSchema.parse({
-        names: searchParams.getAll("names"),
-        usernames: searchParams.getAll("usernames"),
-        title: searchParams.get("title"),
-        meetingProfileName: searchParams.get("meetingProfileName"),
-        meetingImage: searchParams.get("meetingImage"),
-        imageType,
-      });
-
-      const img = new ImageResponse(
-        (
-          <Meeting
-            title={title}
-            profile={{ name: meetingProfileName, image: meetingImage }}
-            users={names.map((name, index) => ({ name, username: usernames[index] }))}
-          />
-        ),
-        ogConfig
-      ) as { body: Buffer };
-
-      return new Response(img.body, { status: 200, headers: { "Content-Type": "image/png" } });
```

Comment:
<img width="781" alt="Screenshot 2024-10-09 at 19 05 56" src="https://github.com/user-attachments/assets/2b75f96e-5e8c-4d3e-9006-38fca476f57a">

According to vercel, new Response + node run time + pages router is not supported. When we fully migrate to app router, we can use new Response here again.

https://vercel.com/docs/functions/og-image-generation#runtime-caveats